Doc/library/ast.rst

```diff
@@ -891,9 +891,13 @@ Statements
    An assignment with a type annotation. ``target`` is a single node and can
    be a :class:`Name`, a :class:`Attribute` or a :class:`Subscript`.
    ``annotation`` is the annotation, such as a :class:`Constant` or :class:`Name`
-   node. ``value`` is a single optional node. ``simple`` is a boolean integer
-   set to True for a :class:`Name` node in ``target`` that do not appear in
-   between parenthesis and are hence pure names and not expressions.
+   node. ``value`` is a single optional node.
+
+   ``simple`` is always either 0 or 1 and represents whether the assignment
+   target is a pure name, and hence simple, or an expression. A target is
+   considered simple if it is a :class:`Name` node that does not appear
+   between parentheses. Only simple targets appear in the :attr:`__annotations__`
+   dictionary of modules and classes.
```

Comment:
It feels unclear what you mean by "pure name" here -- but you go on to describe it clearly in the next sentence. Maybe something like this?

```suggestion
   ``simple`` is always either 0 (indicating a "complex" target) or 1
   (indicating a "simple" target). A "simple" target consists solely of a
   :class:`Name` node that does not appear between parentheses; all other
   targets are considered complex. Only simple targets appear in
   the :attr:`__annotations__` dictionary of modules and classes.
```

Reply:
Thanks! I tried to preserve the term "expression" because if you look at PEP 526, the idea is that "(a): int" is an "expression" and "a: int" is not. But it reads better if we give up on that and just explain the behavior.